docs/reference-client/troubleshooting/node-syncing.md

```diff
@@ -6,6 +6,103 @@ slug: /reference-client/troubleshooting/node-syncing
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Using the Official Database Snapshot Torrent
+
+:::note
+This process assumes you already have chia installed and all files are present in their default locations, to confirm make sure the `~/.chia/mainnet/db/` directory exists.
+
+If you are upgrading from a version prior to Chia 2.0.0, verify your config file has the correct value under the `full_node` section for `database_path` (v1 vs the current v2):
```

Comment:
after `config file` add (~/.chia/mainnet/config/config.yaml)

Reply:
thank you, this is now added